utils/abrt-action-analyze-java.c

```diff
@@ -162,12 +162,11 @@ work_out_list_of_remote_urls(struct sr_java_stacktrace *stacktrace)
         thread = thread->next;
     }
 
-    if (remote_files_csv->buf[0] != '\0')
+    if (remote_files_csv->str[0] != '\0')
```

Comment:
Minor nitpick: Why not just check `len`? It would seem a bit more intuitive to me.